lib/statsd/instrument.rb

```diff
@@ -284,10 +287,24 @@ def measure(key, value = nil, *metric_options, &block)
     end
 
     result = nil
-    value  = 1000 * StatsD::Instrument.duration { result = block.call } if block_given?
+    err = nil
+
+    value = 1000 * StatsD::Instrument.duration {
```

Comment:
Go for `begin .. end` when dealing with multi-line blocks (as stated in https://github.com/bbatsov/ruby-style-guide#syntax).